Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Triggers/InputTypeStateTrigger.bind

```diff
@@ -0,0 +1,46 @@
+<Page
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:triggers="using:Microsoft.Toolkit.Uwp.UI.Triggers"
+    mc:Ignorable="d">
+ <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"  x:Name="root">
```

Comment:
@Kyaa-dost if you're available can you try running this sample on your machine and see if you see the text change when you click or tap on it?

Reply:
@michael-hawker I have tried testing it multiple times in two different machines and it doesn't seem to trigger any action. I still see the same default screen in the SampleApp without any changes.

Reply:
Why don't we just remove it for now? I'm not even clear on what use case this was meant for.

Reply:
Thanks @dotMorten that was going to be my suggestion as well since we couldn't see it working. We can always add it back later if we figure out a practical scenario for it. I think with the compact mode styling WinUI added, there's probably something easier to do there for Desktop vs. Touch.